packages/bottom-tabs/src/__tests__/index.test.tsx

```diff
@@ -70,3 +71,39 @@ test('handles screens preloading', async () => {
     queryByText('Screen B', { includeHiddenElements: true })
   ).not.toBeNull();
 });
+
+test('ignores pointerEvents when hidden', async () => {
+  // @ts-expect-error: incomplete mock for testing
+  jest.spyOn(Animated, 'timing').mockImplementation(() => ({
+    start: (callback) => callback?.({ finished: true }),
+  }));
+  jest.spyOn(keyboardHook, 'useIsKeyboardShown').mockReturnValue(true);
```

Comment:
Okay, done. However, I was unable to do `Keyboard.emit()` since it isn't made available by React Native, and Typescript was complaining.

https://github.com/facebook/react-native/blob/main/packages/react-native/Libraries/Components/Keyboard/Keyboard.js